dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -237,12 +240,17 @@ public Workflow restub(@ApiParam(hidden = true) @Parameter(hidden = true, name =
         Workflow workflow = workflowDAO.findById(workflowId);
         // Check that workflow is valid to restub
         if (workflow.getIsPublished()) {
```

Comment:
It was like this, but `workflow` could be null, which will cause a 5xx. Could be a followup if preferred.